sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/MetricFeedback/MetricFeedbackKind.cs

```diff
@@ -2,31 +2,33 @@
 // Licensed under the MIT License.
 
 using System;
+using Azure.Core;
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
-    /// <summary> The <see cref="FeedbackType"/>. See each specific type for a description of each. </summary>
-    public readonly partial struct FeedbackType : IEquatable<FeedbackType>
+    /// <summary> The <see cref="MetricFeedbackKind"/>. See each specific type for a description of each. </summary>
```

Comment:
should this say `kind` instead?

Reply:
I think in this case it's ok to say type because we're referring to .NET classes.

Reply:
Hmmmm. I think I'll use "kind" anyway.

Reply:
We were not referring to .NET classes. Changed to "kind".